assets/airports/katl.json

```diff
@@ -943,66 +914,133 @@
+                "BBABE": ["BBABE", "LEMKE"],
+                "LEMKE": ["LEMKE"],
+                "MTHEW": ["MTHEW", "MGRIF", "BBABE", "LEMKE"],
+                "RUTTH": ["RUTTH", "JCKEE", "OWNNS"]
             },
-            "body": [["CHOPI", "A150+|S280"], ["ONYON", "A120|S280"]],
+            "body": [],
+            "rwy": {
+                "KATL8L": [["CHPPR", "A130|S250"], "MRCHH", ["^JAGRR", "A90"], "#180"],
+                "KATL8R": [["CHPPR", "A130|S250"], "MRCHH", ["^JAGRR", "A90"], "#180"],
+                "KATL9L": [["CHPPR", "A130|S250"], "MRCHH", ["BSHOP", "A90"], ["NAVVY", "A80+"], ["RYENN", "A70+"], ["AAKAY", "A60+"]],
+                "KATL9R": [["CHPPR", "A130|S250"], "MRCHH", ["BSHOP", "A90"], ["NAVVY", "A80+"], ["ANDIY", "A70+"], ["DFINS", "A60+"]],
+                "KATL10": [["CHPPR", "A130|S250"], "MRCHH", ["BSHOP", "A90"], ["NAVVY", "A80+"], ["^DROYD", "A80"], "#180"],
+                "KATL26L": [["CHPPR", "A130"], "MRCHH", "HIDRO", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL26R": [["CHPPR", "A130"], "MRCHH", "HIDRO", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL27L": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL27R": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL28": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"]
+            },
+            "draw": [
+                []
+            ]
+        },
+        "GLAVN1": {
+            "icao": "GLAVN1",
+            "name": "Glavin One",
+            "entryPoints": {
+                "JKSON": ["JKSON", "MADDX"],
+                "MGRIF": ["MGRIF", "DENST", "AAARN"],
+                "RUSSA": ["RUSSA", "RUTTH", "JKSON", "MADDX"]
+            },
+            "body": ["GLAVN", "STHRN"],
+            "rwy": {
+                "KATL8L": [["YABBA", "A90"], ["NAVVY", "A80+"], ["LARII", "A50+"], ["JAAJJ", "A50+"]],
+                "KATL8R": [["YABBA", "A90"], ["NAVVY", "A80+"], ["GPEAT", "A50+"], ["PEARI", "A50+"]],
+                "KATL9L": [["YABBA", "A90"], ["NAVVY", "A80+"], ["RYENN", "A70+"], ["AAKAY", "A60+"]],
+                "KATL9R": [["YABBA", "A90"], ["NAVVY", "A80+"], ["ANDIY", "A70+"], ["DFINS", "A60+"]],
+                "KATL10": [["YABBA", "A90"], ["NAVVY", "A80+"], ["^DROYD", "A80"], "#180"],
+                "KATL26L": ["HOGGG", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL26R": ["HOGGG", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL27L": ["HOGGG", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL27R": ["HOGGG", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL28": ["HOGGG", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"]
+            },
+            "draw": [
+                []

```

Comment:
Why are we not drawing here? Would be nice for future plans, if STARs are drawn.